app/src/main/res/values/strings.xml

```diff
@@ -388,14 +388,14 @@
     <string name="settings_file_replacement_character_title">Replacement character</string>
     <string name="charset_letters_and_digits">Letters and digits</string>
     <string name="charset_most_special_characters">Most special characters</string>
-    <string name="toast_no_player">No app installed to play this file</string>
+    <string name="toast_no_player">No apps could play this file</string>
```

Comment:
There is no plural after "No":
```suggestion
    <string name="toast_no_player">No app could play this file</string>
```
even better:
```suggestion
    <string name="toast_no_player">No installed app could play this file</string>
```

Reply:
More apps were tried, exhausting the availability of apps installed.
Will change for active request.

Reply:
```suggestion
    <string name="toast_no_player">Install an app that can play the file first</string>
```